build/includes/linux.mk

```diff
@@ -31,13 +31,6 @@ sha_dir = $(shell sha256sum  $(1) | cut -d' ' -f1 | sha256sum | head -c 10 )
 
 # Minikube executable
 MINIKUBE ?= minikube
-# Default minikube driver
-MINIKUBE_DRIVER ?= virtualbox
-# set docker env for minikube
-MINIKUBE_DOCKER_ENV ?= eval $$($(MINIKUBE) docker-env)
-
-# minikube shell mount for certificates
-minikube_cert_mount := ~/.minikube:$(HOME)/.minikube
```

Comment:
This one also might be needed and be different for different OSs. WDYT?

Reply:
For OSX and Linux it's the same.

It _may_ be the same for minikube under WSL, but it requires testing. But I figure we can revisit if we decide it's possible to support windows without hackery.